sos/plugins/pacemaker.py

```diff
@@ -18,22 +18,44 @@
 import os.path
 
 
-class Pacemaker(Plugin, DebianPlugin, UbuntuPlugin):
-    """HA Cluster resource manager"""
+class Pacemaker(Plugin):
+    """Pacemaker high-availability cluster resource manager
+    """
 
     plugin_name = "pacemaker"
+    version = "1.0"
     profiles = ("cluster", )
     packages = (
         "pacemaker",
         "pacemaker-remote",
     )
-    defaults = "/etc/default/pacemaker"
 
     option_list = [
         ("crm_from", "specify the start time for crm_report", "fast", False),
         ("crm_scrub", "enable password scrubbing for crm_report", "", True),
     ]
 
+    envfile = ""
+
+    def setup_crm_mon(self):
+        self.add_cmd_output([
```

Comment:
Ditto for `add_cmd_output()`: you can just pass the "crm_mon" command directly in a single line call.